test/NuGet.Tests.Apex/NuGet.OptProf/Assets/PackageReferenceSdk/PackageReferenceSdk/PackageReferenceSdk.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net7.0</TargetFramework>
+    <TargetFramework>net$(BundledNETCoreAppTargetFrameworkVersion)</TargetFramework>
```

Comment:
I ran `dotnet msbuild -pp:pp.txt`, then opened `pp.txt` in a text editor.

I first searched for `>net8.0<`, and I found one property, but it was in a targets file (so not available in the csproj), plus the property name started with an underscore, which is MSBuild convention for "do not use, it's meant to be internal, but MSBuild doesn't have scopes".

Next I searched for `>8.0<`, and I found `NETCoreAppMaximumVersion` and `BundledNETCoreAppTargetFrameworkVersion`. Interestingly I can't find them in github.com/dotnet/sdk, so I guess it's generated by the build, rather than commit into their source.

In any case, I mentioned it to them, and they said they don't recommend doing this. But in any case OptProf broke because of a change in VS's included SDK, so if another future change breaks it again, maybe we're no worse off than changing the vsconfig file to install the "correct" SDK and runtime for the version of .NET our test project targets. We could add a comment to the csproj saying update the vsconfig when the TargetFramework changes. Maybe that will be less fragile?  For now I'm willing to give this not-recommended way a try.